vscode-test-runner/workspace-1/.joyride/src/integration_test/runner.cljs

```diff
@@ -29,20 +29,27 @@
       (p/resolve! running true)
       (p/reject! running true))))
 
+(defn- run-when-ws-activated [tries]
+  (if (:ws-activated? @db/!state)
+    (do
+      (println "Runner: Workspace activated, running tests")
+      (require '[integration-test.workspace-activate-test])
+      (require '[integration-test.ws-scripts-test])
+      #_(require '[integration-test.require-js-test])
```

Comment:
Activating this require gives a huge trace and causes the tests to never run and VS Code never gets shut down. The run log looks like so:

```
% npm run integration-test

> joyride@0.0.29 preintegration-test
> cd ./vscode-test-runner/workspace-1 && npm i


up to date, audited 2 packages in 398ms

found 0 vulnerabilities

> joyride@0.0.29 integration-test
> node ./vscode-test-runner/launch.js

Found existing install in /Users/pez/Projects/joyride/.vscode-test/vscode-darwin-arm64-insiders. Skipping download
[main 2022-12-31T16:22:53.720Z] update#ctor - updates are disabled by the environment
Electron sandbox mode is enabled!
2022-12-31 17:22:54.199 Code - Insiders Helper (Renderer)[97315:38461317] CoreText note: Client requested name ".NewYork-Regular", it will get TimesNewRomanPSMT rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[NSFont systemFontOfSize:].
2022-12-31 17:22:54.199 Code - Insiders Helper (Renderer)[97315:38461317] CoreText note: Set a breakpoint on CTFontLogSystemFontNameRequest to debug.
Loading development extension at /Users/pez/Projects/joyride
[main 2022-12-31T16:22:56.023Z] UtilityProcess<1>: Creating new...
[main 2022-12-31T16:22:56.025Z] UtilityProcess<1>: received spawn event.
Joyride activate START
Joyride activate END
shadow-cljs - #550 ready!
Hello World, from my-main in user_activate.cljs script
Runner: Workspace not activated yet, tries:  1 - trying again in a jiffy
Runner: Workspace not activated yet, tries:  2 - trying again in a jiffy
Runner: Workspace not activated yet, tries:  3 - trying again in a jiffy
Runner: Workspace not activated yet, tries:  4 - trying again in a jiffy
Runner: Workspace activated, running tests
Cannot find module '../js-file'
Require stack:
- /Users/pez/Projects/joyride/integration_test/require_js_test.cljs: Error: Cannot find module '../js-file'
Require stack:
- /Users/pez/Projects/joyride/integration_test/require_js_test.cljs
    at new cljs$core$ExceptionInfo (/Users/pez/Projects/joyride/out/js/cljs-runtime/cljs/core.cljs:11623:11)
    at Function.cljs$core$IFn$_invoke$arity$3 (/Users/pez/Projects/joyride/out/js/cljs-runtime/cljs/core.cljs:11650:1)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/utils.cljc:134:27
    at Function.cljs$core$IFn$_invoke$arity$4 (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci.impl.utils.js:160:3)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:1197:11
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:126:15
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at Object.sci$impl$interpreter$eval_form [as eval_form] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/interpreter.cljc:40:17)
    at sci$impl$interpreter$eval_string_STAR_ (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/interpreter.cljc:64:23)
    at Object.sci$impl$load$handle_require_libspec [as handle_require_libspec] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:116:30)
    at Function.cljs$core$IFn$_invoke$arity$variadic (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:146:6)
    at Function.cljs$lang$applyTo (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:138:1)
    at Function.cljs$core$IFn$_invoke$arity$4 (/Users/pez/Projects/joyride/out/js/cljs-runtime/cljs/core.cljs:3988:30)
    at Object.sci$impl$load$load_libs [as load_libs] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:186:9)
    at Function.cljs$core$IFn$_invoke$arity$variadic (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:196:4)
    at Function.cljs$lang$applyTo (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/load.cljc:194:1)
    at Function.cljs$core$IFn$_invoke$arity$3 (/Users/pez/Projects/joyride/out/js/cljs-runtime/cljs/core.cljs:3979:30)
    at Function.cljs$core$IFn$_invoke$arity$variadic (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/namespaces.cljc:552:4)
    at sci$impl$namespaces$require (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/namespaces.cljc:551:1)
    at Object.cljs$core$IFn$_invoke$arity$2 (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/lang.cljc:182:5)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:1353:1
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:154:20
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:851:27
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at sci$impl$fns$arity_1 (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/fns.cljc:107:18)
    at /Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/analyzer.cljc:1403:1
    at Object.sci$impl$types$eval [as eval] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/types.cljc:86:8)
    at Timeout.sci$impl$fns$arity_0 [as _onTimeout] (/Users/pez/Projects/joyride/out/js/cljs-runtime/sci/impl/fns.cljc:106:18)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
```